operations/helm/charts/mimir-distributed/ci/offline/test-ingress-values.yaml

```diff
@@ -3,3 +3,6 @@ kubeVersionOverride: "1.20"
 
 ingress:
   enabled: true
+  labels:
+    test: true 
+    type: test
```

Comment:
Sorry, I think I've confused you in the previous suggestion. The PR updates the `gateway.ingress`, and NOT the `ingress`. The correct place to add these test values is `/ci/offline/test-enterprise-values.yaml`.

Could you update the tests and rebuild the assets with `make build-helm-tests`.